src/containers/TexterTodo.jsx

```diff
@@ -182,6 +186,19 @@ export class TexterTodo extends React.Component {
     }
   }
 
+  componentDidMount() {
+    // Get the latest data for the sidebar every 30 seconds to catch new messages
+    if (global.ASSIGNMENT_CONTACTS_SIDEBAR && !this.refreshInterval) {
+      this.refreshInterval = setInterval(this.refreshData, 30000);
+    }
+  }
```

Comment:
@agreenspan24, could you confirm that pagination will assist in the performance levels of this call being requested every 30 seconds?  Just wanting to ensure that this type of call doesn't slow Spoke down.

Reply:
The pagination would help with the performance on the front end- the whole list won't re-render, only a portion of it. We can probably improve the front-end performance slightly, by losing some of the clarity on the pagination, but I don't think it would be worth it.

The server and database, however, will get hit for the list of contacts and statuses every 30 seconds, so that could lead to a slow-down on the server side. I think this is necessary so texters can get to recent replies in a timely manner, but we could up this to every 60 seconds or 3 minutes if we're really worried about server capacity.